pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -1362,14 +1362,23 @@ func (c *Cluster) getEncryptedStatusPVCInitContainer(mountPath string, osdProps
 	   Command successful.
 	*/
 
+	osdResize := `
+	DM_name=%s
+	cryptsetup --verbose status "$DM_name"
+	ceph osd crush reweight osd.0 1
```

Comment:
Is this supposed to only be osd.0?

Reply:
This was prototyping, we are going a different direction now.